packages/adapters/gupshup-whatsapp/src/gupshupWhatsappAdapter.spec.ts

```
@@ -314,14 +327,124 @@ describe('gupshup whatsapp adapter', () => {
+      },
+      transformer: {
+        metaData: {}  // We'll ignore sessionId in comparison
       }
     };
     const xmsg = await adapter.convertMessageToXMsg(mockAudioMessage);
     expect(xmsg.messageId.Id).toBeDefined();
     xmsg.messageId.Id = 'testId';
     expect(xmsg.timestamp).toBeGreaterThan(0);
     xmsg.timestamp = 0;
-    xmsg.messageId.Id = 'testId';
+    delete xmsg.transformer?.metaData?.sessionId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)